### DISCUSSION TITLE
Wildcard  issue: Dns01Precondition failed The delegated name server is not correct

### DISCUSSION THREAD

**Discussion by meps007** [shibayan/appservice-acmebot#287] [Q&A] [answered]
Hi, after following your detailed installation guide I've manged to update my non-wildcard domein names. The wildcard domein name appears in the add-certificate Deployment slot, but when submitting I get the error  " "The activity function 'Dns01Precondition' failed: "The delegated name server is not correct. DNS zone , Name Servers = ns1-05.azure-dns.com......" 

How can I renew mine wildcard certificate?

**Comment by shibayan** [comment, marked as answer]
The domain you are trying to use is not configured for Azure DNS. You will need to configure NS for your domain.

https://docs.microsoft.com/en-us/azure/dns/dns-delegate-domain-azure-dns#delegate-the-domain
